www/getstarted.json

```diff
@@ -24,7 +24,7 @@
         "classes": "xs-hide sm-hide",
         "href": "/"
       },
-      "name": "The Blog",
+      "name": "AMP start home logo",
```

Comment:
This should have been on alt property that you were supposed to add to the logo right?